content/news/2025-01-22_rocur-relaunch/index.en.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "We're Launching the R-Ladies Rotating Curation (RoCur) Account on Bluesky!"
+author: "RoCur team"
+type: blog
+date: "2025-01-22"
+image: img/bluesky_media_kit_logo_1.png
+tags: 
+- rocur
+categories: 
+- R-Ladies
+summary: We're looking for curators for the We Are R-Ladies Bluesky account!
+slug: rocur-relaunch-2025
+---
+
+![Bluesky logo. A white butterfly on a blue gradient.](img/bluesky_media_kit_logo_1.png)
+
+**Caption:** Bluesky logo, sourced from the official [Bluesky media kit](https://drive.google.com/drive/folders/16mlJPfWNnc6jj-3vGZ88SFysIY13GBp0).
+
+We're absolutely thrilled to announce that the [R-Ladies Rotating Curation (RoCur) account](https://bsky.app/profile/weare.rladies.org) is now live on Bluesky! Each week, a new member of the R-Ladies community will take over the account to talk about all things R. This will be a fantastic opportunity for individuals who identify as a minority gender in the R community to amplify their voices and connect with others in the greater data science community.
+
+## History
+
+Some of you might recall the RoCur account on Twitter. The program ran for about five years, giving over 100 curators the chance to take over the account and share their stories. While the Twitter account is no longer active, the archive of posts can still be viewed [here](https://rladies.org/tweet-archive/wearerladies/).
+
+## How You Can Help
+
+We're excited to revive the RoCur program on Bluesky, picking up where we left off. You can help us build back the community stronger than ever by signing up to be a curator or nominating someone to be a curator!
+
+## Become a Curator
+
+*Are you interested in becoming a curator?* Curating is a great way to build your social media presence, develop your science communication skills, expand your CV, and connect with a supportive community. Sign up to be a curator on [this form](https://airtable.com/appjkZZgtF0iEWFjx/pagqsAma1WmUYxa9j/form). We're currently looking for curators until the middle of 2025, and we'll open up more spots as the year progresses.
```

Comment:
```suggestion
*Are you interested in becoming a curator?* Curating is a great way to build your social media presence, develop your science communication skills, expand your CV, and connect with a supportive community. Sign up to be a curator on the [@weare.rladies.org Curator Sign Up form](https://airtable.com/appjkZZgtF0iEWFjx/pagqsAma1WmUYxa9j/form). We are currently looking for curators until the middle of 2025, and we will open up more spots as the year progresses.
```

Reply:
also, I'm not sure about the "expand your CV" part...

you could also add something about learning and sharing